base/math.jl

```diff
@@ -776,9 +776,10 @@ end
 ldexp(x::Float16, q::Integer) = Float16(ldexp(Float32(x), q))
 
 """
-    exponent(x) -> Int
+    exponent(x::BigFloat)
+    exponent(x::T) where T<:Union{Float16, Float32, Float64}
```

Comment:
This would be clearer in the original form just putting an `::AbstractFloat` annotation on `x`, i.e.
```jl
exponent(x::AbstractFloat) -> Int
```

Reply:
I did the changes